packages/contracts/contracts/Interfaces/ITroveManager.sol

```diff
@@ -11,6 +11,14 @@ import "./IPCV.sol";
 // Common interface for the Trove Manager.
 interface ITroveManager is ILiquityBase {
 
+    enum Status {
```

Comment:
Shouldn't be used this enum in BorrowedOperations.sol line #213?

```
        // Set the trove struct's properties
        contractsCache.troveManager.setTroveStatus(msg.sender, 1);
```

Reply:
Thanks! Fixed in #93 PR, this commit https://github.com/Threshold-USD/dev/pull/93/commits/003f498467bf2943483f63c363e49143bf51e02b